web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/ytelsefordeling/FørsteUttaksdatoTjenesteImpl.java

```diff
@@ -47,25 +44,16 @@
         var oppgittePerioder = ytelseFordelingTjeneste.hentAggregat(behandling.getId())
             .getGjeldendeFordeling()
             .getPerioder();
-        if (oppgittePerioder.isEmpty()) {
-            return Optional.empty();
-        }
-        return Optional.of(sortert(oppgittePerioder).get(0).getFom());
+        return oppgittePerioder.stream().map(op -> op.getFom()).min(LocalDate::compareTo);
     }
 
     private Optional<LocalDate> finnFørsteUttaksdatoRevurdering(Behandling behandling) {
         var revurderingId = behandling.getOriginalBehandlingId()
             .orElseThrow(() -> new IllegalStateException("Utviklerfeil: Original behandling mangler på revurdering - skal ikke skje"));
         var uttak = uttakTjeneste.hentUttakHvisEksisterer(revurderingId);
```

Comment:
Hm, bør du ikke gjøre som i STP, velge min(fudForrigeUR, fudDenneYF) ?

Reply:
Denne er veldig spesifikk til validering i overstyring av fakta om uttak. Flytter koden inn i dto/oppdaterer tjenesten for å vise dette